src/translations/translationMetadata.json

```diff
@@ -1,9 +1,12 @@
 {
+	"ar": {
+		"nativeName": "العربية"
+	},
 	"bg": {
 		"nativeName": "Български"
 	},
-	"bn": {
```

Comment:
Why are you removing Bengali?

Reply:
I thought it was requested with the along other Indian languages by skalavala, but I was wrong.
Also sobody submitted translations for it.